workflows/edit_data_packages/add_a_pre-generated_identifier.Rmd

```diff
@@ -10,4 +10,4 @@ eml@packageId <- new("xml_attribute",
                       .Data = doiPid)
 ```
 
-Be sure to include the `identifier=` argument in your `publish_update` command so the pre generated identifier is applied.
+Be sure to include the `identifier=` argument in your [publish update](#publish-update) command so the pre generated identifier is applied.
```

Comment:
The cross-link looks good to me. You won't be able to see it work until it gets knit together into the bookdown though.